tests/index.spec.js

```diff
@@ -392,6 +392,53 @@ describe('Testing generic actions', () => {
     expect(result.fee_collected).to.be.a('number')
   })
 
+  it(`RE Add public addresses`, async () => {
+    const result = await fioSdk.genericAction('addPublicAddresses', {
+      fioAddress: newFioAddress,
+      publicAddresses: [
+        {
+          chain_code: ethChainCode,
+          token_code: ethTokenCode,
+          public_address: 'xxxxxxyyyyyyzzzzzz1',
+        },
+        {
```

Comment:
I think remove fio pub address from this array

Reply:
ill do this..